### ISSUE TITLE
v0.14.10 release checklist

### ISSUE THREAD

**Issue by alerque** [sile-typesetter/sile#1830]
- [x] Shuffle any issues being put off to future milestones
- [x] Close all issues in current milestone (except this one)
- [x] Spring clean
    - [x] Re-fetch tooling
- [x] Configure and build
- [x] Pass all tests
    - [x] Remote CI
    - [x] Local
- [x] Cut release
- [x] Push release commit & tag to master
- Update website
    - [x] Copy and post manual, update 'latest' symlink and menu links
    - [x] Copy changelog and prefix with a summary as a blog post
    - [x] Tweak summary for gfm and edit into GitHub release notes
- Update downstream distro packages
	- [x] Arch Linux official: [community/sile](https://archlinux.org/packages/community/x86_64/sile/)
	- [x] Arch Linux AUR: [AUR/sile-luajit](https://aur.archlinux.org/packages/sile-luajit) 
    - [x] Homebrew: [Formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/sile.rb) – [pull request](https://github.com/Homebrew/homebrew-core/pull/136324)
    - [x] NixOS: [nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/typesetting/sile/default.nix) – [pull request](https://github.com/NixOS/nixpkgs/pull/242817) – [tracker](https://nixpk.gs/pr-tracker.html?pr=242817)
        - [x] Drop devel flag from any website examples that were using it
    - [x] Ubuntu: [ppa](https://launchpad.net/~sile-typesetter/+archive/ubuntu/sile)
    - [x] Docker Hub: [tags](https://hub.docker.com/repository/docker/siletypesetter/sile/tags)
    - [x] GHCR: [versions](https://github.com/orgs/sile-typesetter/packages/container/sile/versions)
    - [ ] openSUSE: pinging @Pi-Cla re [specfile](https://build.opensuse.org/package/show/openSUSE%3AFactory/sile)
    - [ ] Fedora: pinging @hegjon re [copr](https://copr.fedorainfracloud.org/coprs/jonny/SILE/)
    - [ ] NetBSD: pinging @jsonn re [pkgsrc](https://pkgsrc.se/print/sile)
    - [ ] Void Linux: [pull request](https://github.com/void-linux/void-packages/pull/18306)
    - [ ] OpenBSD: [mailing list thread](https://marc.info/?t=157907840200001)
- Bump downstream projects
    - [x] [FontProof](https://github.com/sile-typesetter/fontproof) (Docker image base plus CI matrix)
    - [x] [CaSILE](https://github.com/sile-typesetter/casile) (check, consider patch release)
- [ ] Eat cake

[![Packaging status](https://repology.org/badge/vertical-allrepos/sile.svg)](https://repology.org/project/sile/versions)

Previous checklist: #1772 ([release](https://github.com/sile-typesetter/sile/releases/tag/v0.14.9))

**Comment by hegjon** [comment]
> Going to wait for package maintainers to accept my updates to some lua dependencies before I update the SILE package.

What lua dependencies needs to be updated?

**Comment by Pi-Cla** [comment]
> > Going to wait for package maintainers to accept my updates to some lua dependencies before I update the SILE package.
> 
> What lua dependencies needs to be updated?

Oh, they actually accepted them very quickly. SILE package should be updated very soon on openSUSE

**Comment by hegjon** [comment]
Do you have a list? I probably need to bump dependencies for Fedora

**Comment by Pi-Cla** [comment]
> Do you have a list? I probably need to bump dependencies for Fedora

lua expat, lua sec, and lua lpeg. You can check [this](https://archlinux.org/packages/?q=lua&sort=-last_update)  for any other dependency updates that Arch has already done

**Comment by hegjon** [comment]
Btw, I just added this @Pi-Cla :

```
Requires: libtexpdf = %{version}
```

Due to the bundling and keeping the same version at runtime.

**Comment by hegjon** [comment]
I have updated the Fedora Linux COPR project https://copr.fedorainfracloud.org/coprs/jonny/SILE/build/6167467/

**Comment by alerque** [comment]
There haven't been any changes to the dependency list or minimum required versions thereof. There do happen to have been some upstream updates in the Lua ecosystem with nice things and it would be good to get them bumped to the latest versions for the sake of end users, but SILE itself doesn't care.

**Comment by alerque** [comment]
@bumbleblue Please do file a separate issue about that. If you're talking about the `book.sil` example posted in our website example repos, I just test built it with v0.14.10 and didn't have a problem, so some more details will be needed to figure out what issue you encountered. Happy to help this issue just isn't the place for it and I'll need a few more exact details to go on.
